x-pack/legacy/plugins/code/server/log.ts

```diff
@@ -84,9 +83,9 @@ export class Logger implements VsLogger {
       });
     }
     if (this.verbose) {
-      this.server.log([...this.baseTags, 'info', 'stdout'], msg);
+      this.logger.get(...this.baseTags).info(msg);
```

Comment:
LoggerFactory doesn't re-create Logger object for every `get `call, but probably we can call `get` once in a constructor? In this case, we don't rely on this implementation detail.

Reply:
done